data/routers/register-router.js

```diff
@@ -0,0 +1,42 @@
+const router = require('express').Router();
+
+const Register = require('../models/register-model');
+const Users = require('../models/users-model');
+
+router.post('/', async (req, res) => {
+  const { username, password } = req.body;
+  if (!username || !password) {
+    res.status(400).json({
+      error: 'You must provide a username and password to register.'
+    });
+  } else if (username.length > 64 || password.length > 128) {
+    res.status(400).json({
+      error:
+        'Your username may not exceed 64 characters. Your password may not exceed 128 characters.'
+    });
```

Comment:
take a look at doing some form validation using the Joi library -- you can set specific requirements like this by just passing the req.body into a Joi function you make which can be reused in any endpoint you need to check the same info for